modules/profilereview/src/Auth/Process/ProfileReview.php

```diff
@@ -148,36 +148,36 @@ protected function getAttributeAllValues($attributeName, $state)
      * @returns string
      * @return mixed|string
      */
-    protected static function getRelayStateUrl($state)
+    protected static function getRelayStateUrl(array $state): mixed
     {
         if (array_key_exists('saml:RelayState', $state)) {
             $samlRelayState = $state['saml:RelayState'];
 
-            if (strpos($samlRelayState, "http://") === 0) {
+            if (str_starts_with($samlRelayState, "http://")) {
                 return $samlRelayState;
             }
 
-            if (strpos($samlRelayState, "https://") === 0) {
+            if (str_starts_with($samlRelayState, "https://")) {
                 return $samlRelayState;
             }
         }
         return '';
     }
 
-    protected function initComposerAutoloader()
+    protected function initComposerAutoloader(): void
```

Comment:
That checking of a function's type definitions against a library's overridden function includes the return type.